docs/get-started-with-datahub.md

```diff
@@ -1,63 +1,21 @@
 # Get Started With DataHub
 
-This section is focused on helping you get up and running with DataHub as quickly as possible.
-
-## Get Started with Self-Hosted DataHub
-
-This is a great place to start if you're new to DataHub and are interested in taking the open-source project for a spin.
```

Comment:
Per our convo: replace this with a brief "self-hosted datahub is the OSS implementation of our tool,etc.." just a small blurb about what you are clicking into, maybe a persona snippet like "if you're looking to build and deploy datahub yourself, start here" 

then BUTTON